cmd/mysterium_client/run/command.go

```diff
@@ -98,3 +105,13 @@ func (cmd *CommandRun) Kill() error {
 
 	return nil
 }
+
+func sigtermListener(terminator chan os.Signal, cmd *CommandRun) {
```

Comment:
Function should be named as a verb, not as a noun - maybe `listenSigterm` suites better?